src/frontend/frontend/console.js

```diff
@@ -213,47 +214,53 @@ angular.module('frontend-app')
     socket.register_callback("io", function(io) {
       if(io.type == "stdout") {
         ind = io.message.indexOf("\n");
-        if(ind > -1) {
+        if (ind > -1) {
```

Comment:
You should probably remove all the parse_asan code now that we don't need/use it here.